nucypher/characters/lawful.py

```diff
@@ -853,6 +853,29 @@ def __substantiate_stamp(self):
         message = f"Created decentralized identity evidence: {self.__decentralized_identity_evidence[:10].hex()}"
         self.log.debug(message)
 
+    @property
+    def decentralized_identity_evidence(self):
+        return self.__decentralized_identity_evidence
+
+    @property
+    def operator_address(self):
+        if not self.federated_only:
+            # TODO: The reason for the fork here is the difference in available information
```

Comment:
Definitely an awkward branching and getter logic here.  Deserves an issue to track.

Reply:
Filed https://github.com/nucypher/nucypher/issues/2875